### ISSUE TITLE
hide "Eintrag hinzufügen" for normal users?

### ISSUE THREAD

**Issue by simonkoehl** [future-coop/transition-map#5]
and provide a respective URL for editors

**Comment by sonja-retzlaff** [comment]
I agree. Especially as long as everybody can add reviews regarding sustainability criteria. The map is already index in search engines and currently everybody can just add new entries. In order to ensure quality of entries, I´d only make adding and editing entries option available to Transition Map staff.

![Transition Map Ecosia Search](https://user-images.githubusercontent.com/47123520/54197188-932ca980-44c3-11e9-8f1a-d820e4dbde4b.JPG)
